packages/sources/oanda/src/transport/modifiedSSE.ts

```diff
@@ -29,6 +29,7 @@ export type ModifiedSSEConfig = {
 }
 
 export class ModifiedSseTransport<T extends TransportGenerics> extends StreamingTransport<T> {
+  name = 'default_single_transport'
```

Comment:
I think this one should be `sse` like it was before since this adapter has a router